vaadin-markdown-flow-parent/vaadin-markdown-flow/src/main/java/com/vaadin/flow/component/markdown/Markdown.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2000-2025 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.component.markdown;
+
+import java.util.Objects;
+
+import com.vaadin.flow.component.Component;
+import com.vaadin.flow.component.HasSize;
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
+
+/**
+ * Markdown is a component for rendering Markdown content. It takes Markdown
+ * source as input and renders the corresponding HTML.
+ * 
+ * @author Vaadin Ltd
+ */
+@Tag("vaadin-markdown")
+@NpmPackage(value = "@vaadin/polymer-legacy-adapter", version = "24.8.0-alpha17")
+@JsModule("@vaadin/polymer-legacy-adapter/style-modules.js")
+@NpmPackage(value = "@vaadin/markdown", version = "24.8.0-alpha17")
+@JsModule("@vaadin/markdown/src/vaadin-markdown.js")
+public class Markdown extends Component implements HasSize {
+
+    private String serverContent;
+    private String clientContent;
+
+    /**
+     * Default constructor. Creates an empty Markdown.
+     */
+    public Markdown() {
+    }
+
+    /**
+     * Creates a Markdown with content.
+     *
+     * @param markdown
```

Comment:
The name doesn't match with the actual parameter name. Same for the other methods below.

Reply:
Oops, the property was renamed an I missed to update these. Done.